File: nivel-00/o-dobro-de-um-numero-qualquer/foo.js
var assert = require('assert');

//
// Função que retorna o dobro de um número qualquer
//
var dobro = function(num) {
    return num * 2;
}

//
// Testes
//
try {
    assert.equal(10, dobro(5));
} catch(e) {
    console.log(e);
}

File: nivel-00/o-dobro-de-um-numero-qualquer/foo.c
#include <stdio.h>
#include <assert.h>

//
// Função que retorna o dobro de um número qualquer
//
int dobro(int num) {
    return 2 * num; // <------- olha aqui o resultado !!!!!!!!!!
}

//
// Testes
//
int main (){

    assert(10 == dobro(5));
    assert(12 == dobro(6));

    return 0;
}
